launch_testing/launch_testing_examples/package.xml

```diff
@@ -13,18 +13,20 @@
   <author email="aditya.pande@openrobotics.org">Aditya Pande</author>
   <author email="sloretz@openrobotics.org">Shane Loretz</author>
 
+  <exec_depend>launch</exec_depend>
+  <exec_depend>launch_ros</exec_depend>
+  <exec_depend>launch_testing</exec_depend>
+  <exec_depend>launch_testing_ros</exec_depend>
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>rcl_interfaces</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
+
   <test_depend>ament_copyright</test_depend>
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pytest</test_depend>
```

Comment:
This should also be an `exec_depend`, not a `test_depend`.

Reply:
To clarify, `python3-pytest` seems to be listed as `test_depend` in several other `ament_python` packages in `ros2` org.

Reply:
Yeah.  The difference in this package is that `pytest` is used in the actual code of the package, not in the tests (that is, the code that does `import pytest` is in the `launch_testing_examples` subdirectory, not the `test` subdirectory).  Thus, these should all be `exec_depend`.

Reply:
Got it 👍🏼